gh-129169: Update docs on asyncio.ensure_future() to make it more clear that a reference of the returned value should be stored and provide an alternative to avoid this. (#129986)
python/cpython: Doc/library/asyncio-future.rst

   .. important::

      Save a reference to the result of this function, to avoid
      a task disappearing mid-execution.

>       See also the :func:`create_task` function which is the
>       preferred way for creating new Tasks or
>       :meth:`asyncio.TaskGroup.create_task`, an alternative
>       which keeps the reference within the :class:`asyncio.TaskGroup`.
> 

Reviewer: ```suggestion
      See also the :func:`create_task` function which is the
      preferred way for creating new tasks or use :class:`asyncio.TaskGroup`
      which keeps reference to the task internally.

```
